release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2478,6 +2478,196 @@ index a90ab14ff..f7f6f1a71 100644
+----
+
++
+Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])
+
+* During a primary node replacement in a bare-metal cluster, the new primary host gets stuck in a `Provisioning` state, but the node reports into the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])
+
+* Using {rh-rhacm-first}, spoke cluster deployments on Dell PowerEdge R640 servers are blocked when the virtual media does not disconnect the ISO in the iDRAC console after writing the image to the disk. To workaround this issue, disconnect the ISO manually through the Virtual Media tab in the iDRAC console. (link:https://issues.redhat.com/browse/OCPBUGSM-45884[*OCPBUGSM-45884*])
+
+* On a dual-stack networking environment, devices and connections get stuck in an `ip-check` state with a default `dhcp6` profile generated by the `nm-initrd-generator` utility. The `/etc/resolve.conf` file is not generated. To workaround this issue, restart the `NetworkManager` service. This generates the missing `/etc/resolve.conf` file and the installation can continue. (link:https://issues.redhat.com/browse/RHELPLAN-127788[*RHELPLAN-127788*], link:https://issues.redhat.com/browse/OCPBUGS-70[*OCPBUGS-70*])
+
+* Using NVIDIA-branded Mellanox NICs on Dell hardware, incoming packets that are larger than the preset F5 application receive buffer (currently set to 8K) arrive with an incorrect VLAN tag. This results in sporadic truncated packets being delivered by the NIC. (link:https://issues.redhat.com/browse/RHELPLAN-123058[*RHELPLAN-123058*])
+
+* Sometimes, a {sno} node configured with static IP and deployed using the GitOps ZTP pipeline becomes unreachable during Day 2 operator configuration. The {product-title} cluster installation completes successfully, and the cluster is healthy. After reboot, the node becomes unreachable with the network interface down. (link:https://issues.redhat.com/browse/OCPBUGSM-46688[*OCPBUGSM-46688*])
+
+* After removing the `SriovNetworkNodePolicy` policy from the Git repository, the `SriovNetworkNodePolicy` resource managed by the removed policy remains on the spoke cluster. (link:https://issues.redhat.com/browse/OCPBUGSM-34614[*OCPBUGSM-34614*])
+
+* When upgrading the PTP Operator from 4.10 to 4.11, the OpenShift subscription reports the error 'no channel heads (entries not replaced by another entry) found in channel `"stable"` of package `"ptp-operator"`. However, the Operator upgrades successfully. (link:https://issues.redhat.com/browse/OCPBUGSM-46114[*OCPBUGSM-46114*])
```

Comment:
The 
'no channel heads (entries not replaced by another entry) found in channel `"stable"` of package `"ptp-operator"`
has a single quote opening and no closing punctuation.